test/MeshKernelNETTest/Api/MeshKernelTest.cs

```diff
@@ -156,6 +156,67 @@ public void Mesh2dInsertEdgeThroughApi()
             }
         }
 
+        [Test]
+        public void Mesh2dInsertEdgeFromCoordinatesThroughApi()

```

Comment:
Would be nice to also have a test with inserting an edge with only one coordinate matching an existing vertex, and one with no coordinate matching an existing vertex.